src/components/HeadCellMenu/HeadCellMenu.tsx

```diff
@@ -76,6 +76,7 @@ const HeadCellMenu = ({
         aria-expanded={open ? 'true' : undefined}
         aria-haspopup="true"
         data-testid="nebula-table-utils-head-menu-button"
+        isInteractionsActive={searchRelatedArgs?.interactions.active ?? false}
```

Comment:
Maybe it time to move `interactions` out from `searchRelatedArgs`? As it defines if the menu can be opened or not

Reply:
Sure, sounds legit, I'll create a PR for that in future